contracts/rewards_fuses/morpho/MorphoClaimFuse.sol

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.26;
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import {PlasmaVaultLib} from "../../libraries/PlasmaVaultLib.sol";
+import {PlasmaVaultConfigLib} from "../../libraries/PlasmaVaultConfigLib.sol";
+import {IUniversalRewardsDistributor} from "./ext/IUniversalRewardsDistributor.sol";
+
+/**
+ * @title MorphoClaimFuse
+ * @notice A fuse contract responsible for claiming rewards from Morpho's UniversalRewardsDistributor
+ * @dev This contract acts as an intermediary to claim rewards and forward them to the RewardsClaimManager
+ */
+contract MorphoClaimFuse {
+    using SafeERC20 for IERC20;
+
+    /**
+     * @notice Emitted when rewards are successfully claimed from the UniversalRewardsDistributor
+     * @param version Address of this contract instance
+     * @param rewardsToken Address of the token being claimed as rewards
+     * @param rewardsTokenAmount Amount of rewards tokens claimed
+     * @param rewardsClaimManager Address of the RewardsClaimManager receiving the claimed rewards
+     */
+    event MorphoClaimFuseRewardsClaimed(
+        address version,
+        address rewardsToken,
+        uint256 rewardsTokenAmount,
+        address rewardsClaimManager
```

Comment:
why rewardsClaimManager is a part of event?

Reply:
Since I wanted to know where the funds were delivered after the event.